projects/rocprofiler-sdk/source/share/rocprofiler-sdk-rocpd/data_views.sql

```diff
@@ -315,7 +315,7 @@ SELECT
     K.workgroup_size_z AS workgroup_z,
     K.group_segment_size AS lds_size,
     K.private_segment_size AS scratch_size,
-    S.arch_vgpr_count AS vgpr_count,
+    S.arch_vgpr_count,
```

Comment:
```suggestion
    S.arch_vgpr_count AS vgpr_count,
```
Same, you probably need to revert these back:
https://github.com/ROCm/rocm-systems/commit/ea891a16fdba9659aa577f8a18fdbeb7da7cab58

Reply:
Looks like change made.